apps/web/pages/signup.tsx

```diff
@@ -57,9 +57,10 @@ export default function Signup({ prepopulateFormValues, token, orgSlug, orgAutoA
   const telemetry = useTelemetry();
   const { t, i18n } = useLocale();
   const flags = useFlagMap();
+  const isOrgInviteByLink = orgSlug && !prepopulateFormValues?.username;
   const methods = useForm<FormValues>({
     mode: "onChange",
-    resolver: zodResolver(signupSchema),
+    resolver: zodResolver(isOrgInviteByLink ? signupSchema.omit({ username: true }) : signupSchema),
```

Comment:
Fixing bug that validated existence of a username in a org context when it's not being inputed by the user as it is inferred.

Reply:
I couldn't replicate the bug. In my testing username was automatically computed and thus was available.

Tried with both an email that matches and not matches autoAcceptDomain